IFC4x3/Properties/p/PantographType_1qXzLtDkHAO9qk9H6cFVer/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the type of pantograph as a desgin parameter for the overhead contactline system.
```

Comment:
Would it make sense to use `Object Typing`?

Reply:
This is design property for the OCL system, not really a part of the OCL system and will not be in BIM model, so I think we need to keep it.